packages/zoe/test/unitTests/contracts/test-priceAggregatorChainlink.js

```diff
@@ -175,9 +181,9 @@ test('basic', async t => {
 
   // ----- round 1: basic consensus
   await oracleTimer.tick();
-  await E(pricePushAdminA).pushResult({ roundId: 1, data: '100' });
-  await E(pricePushAdminB).pushResult({ roundId: 1, data: '200' });
-  await E(pricePushAdminC).pushResult({ roundId: 1, data: '300' });
+  await E(pricePushAdminA).pushResult({ roundId: 1, unitPrice: 100n });
+  await E(pricePushAdminB).pushResult({ roundId: 1, unitPrice: 200n });
+  await E(pricePushAdminC).pushResult({ roundId: 1, unitPrice: 300n });
   await oracleTimer.tick();
 
   const round1Attempt1 = await E(aggregator.creatorFacet).getRoundData(1);
```

Comment:
100, 200, and 300 seem pretty far apart; does the aggregator consume them no matter how far apart they are? Or is there a point where it says "sources don't agree; nothing available at this time"?

Reply:
Right, it lets the prices push as long as they're within the `minSubmissionValue` and `maxSubmissionValue`. Extremity of gap doesn't matter because the feed only exposes the median.

Reply:
I'm missing something about how scale works in this contract. DAI has 18 decimals. the `.value` of an ERTP amount of 1 DAI is bigger than 100K. How could this possibly work?